DXMainClient/DXGUI/Multiplayer/GameLobby/SkirmishLobby.cs

```diff
@@ -57,6 +58,21 @@ public override void Initialize()
             ProgramConstants.PlayerNameChanged += ProgramConstants_PlayerNameChanged;
         }
 
+        protected override void OnEnabledChanged(object sender, EventArgs args)
+        {
+            base.OnEnabledChanged(sender, args);
+            if (Enabled)
+            {
+                ddPlayerSides[0].SelectedIndexChanged += PlayerSideChanged;
+                UpdateDiscordPresence(true);
+            }
+            else
+            {
+                ddPlayerSides[0].SelectedIndexChanged -= PlayerSideChanged;
+                ResetDiscordPresence();
+            }       
+        }
```

Comment:
What is the point of ever unsubscribing this? It shouldn't cause any harm to be subscribed to it at all times, so you could remove this entire override.